### ISSUE TITLE
Big pack of files that cause problems

### ISSUE THREAD

**Issue by qarmin** [astral-sh/ruff#3721]
Here I will public packs of broken files(mostly autofix bugs), that I found when fuzzing project.
I know that such issues should be created in separate issues, but I don't really have time to do this(there is ~700 broken files, which most probably have same source of problems), so feel free to do report it if you want.

I will try to provide new pack of  broken files for each release

Command which I used
```
ruff . --fix --config ruff.conf
```
ruff.conf (please comment, if there is better way to test even more rules, or get better coverage of project)
```
select = ["ALL"]
exclude = []
```

A lot of files is minimized, so I suggest to check first the smallest files(a lot of files have less than 1KB ).
0.0.259 - 713 files - 1bac206995043d7d533338b7f8b7db37f37bf43b - [Broken_259_713.zip](https://github.com/charliermarsh/ruff/files/11066017/Broken_259_713.zip)
0.0.260 - 962 files - a142d71e0b4c673b05647a0ac6aeb4754e8cca48 - [Broken_260_962.zip](https://github.com/charliermarsh/ruff/files/11113310/Broken_260_962.zip)
0.0.261 - 767 files - cae5503e348ecce139aab66c2d6166087951d7ff - [Broken_261_767.zip](https://github.com/charliermarsh/ruff/files/11173660/Broken_261_767.zip)
0.0.262 - 1384 files - 6e8d561090f50b5c1a62618f4ee9de68f48017ef - [Broken_262_1384.zip](https://github.com/charliermarsh/ruff/files/11287601/Broken_262_1384.zip)
0.0.263 - 3678 files - 3c9f5e2fdcce32d56515da1063e0fd23e5d7c215 - [Broken_263_3678.zip](https://github.com/charliermarsh/ruff/files/11340258/Broken_263_3678.zip)
0.0.265 - 5417 files - 11e1380df412cd18eb8f6001e6164195bf90436b - [Broken_265_5417.zip](https://github.com/charliermarsh/ruff/files/11409007/Broken_265_5417.zip)
0.0.285 - 374 files - ea72d5feba369b7be2c49302557620b2788beb85 - [Broken_285_374.zip](https://github.com/astral-sh/ruff/files/12393621/Broken_285_374.zip)

**Comment by charliermarsh** [comment]
Thank you! This is great, no need to file separate issues.

**Comment by jfmengels** [comment]
@qarmin can I ask how you generated these fuzzed files? I'd be interested for doing the same for my own linter (elm-review, for Elm).

**Comment by charliermarsh** [comment]
@jfmengels -- I believe it was done with `cargo-fuzz`, see https://github.com/charliermarsh/ruff/issues/3507.

**Comment by qarmin** [comment]
https://crates.io/crates/create_broken_files - tool to create broken files

https://github.com/qarmin/Automated-Fuzzer - wrapper on this tool, that automates creating and checking broken files

**Comment by charliermarsh** [comment]
We love broken files

**Comment by qarmin** [comment]
Closing since I managed to report them one by one without too much duplicates
